openshift/e2e-common.sh

```diff
@@ -132,11 +144,23 @@ function run_e2e_tests(){
       local run_command="-run ^(${test_name})$"
   fi
 
-  go_test_e2e -timeout=50m -parallel=20 ./test/e2e \
+  # check for test flags
+  RUN_FLAGS=("-timeout=50m -parallel=20")
+  if [ -n "${EVENTING_TEST_FLAGS:-}" ]; then
+    RUN_FLAGS="${EVENTING_TEST_FLAGS}"
+  fi
+
+  # check for test args
+  if [ -n "${EVENTING_TEST_ARGS:-}" ]; then
+    common_opts=("${EVENTING_TEST_ARGS}")
```

Comment:
You're assigning an array to a variable that was defined as "string" earlier. Perhaps better to use just `common_opts="${EVENTING_TEST_ARGS}"`